client_render.go

```diff
@@ -27,6 +34,8 @@ type ClientRenderer struct {
 	//
 	// Defaults to "pre".
 	ContainerTag string
+
+	initializeOptions initializationOptions
```

Comment:
Putting initializationOptions here wouldn't work. It would make it unusable for a user. your original version where Theme was a field here is the right way to do it: Theme field that we then build initializationOptions from.